### ISSUE TITLE
Error validating credentials.

### ISSUE THREAD

**Issue by nsantini** [Microsoft/BotFramework-Emulator#258]
When I run the emulator built locally (by cloning this repo) I get an error on the getAccessToken step, the error is the same that I get when I try to get the access token following the instructions on https://blog.botframework.com/2017/06/19/Load-Testing-A-Bot/
The error is:
```
{
  "error": "server_error",
  "error_description": "AADSTS70002: Error validating credentials.\r\nTrace ID: f86e6fc8-2beb-44d7-b4bf-880951871c00\r\nCorrelation ID: ca294946-1a3b-4dcc-915c-faddbe8fbf66\r\nTimestamp: 2017-07-31 02:51:35Z",
  "error_codes": [
    70002
  ],
  "timestamp": "2017-07-31 02:51:35Z",
  "trace_id": "f86e6fc8-2beb-44d7-b4bf-880951871c00",
  "correlation_id": "ca294946-1a3b-4dcc-915c-faddbe8fbf66"
}
```

**Comment by eanders-MS** [comment]
Hi @nsantini, We're working to resolve this issue. Commit 4923221cb52fa60c586094db81405a3ef10955a5 is the last known good.

**Comment by nsantini** [comment]
@eanders-MS that commit seems to be related to speech support, the issue I mentioned happens on conversationManager.ts

Unless there are previous commits on that version that solve the issue?

**Comment by eanders-MS** [comment]
The commit I mentioned is the one immediately preceding the commit that breaks authentication. If you sync to that version of the codebase, you should be able to talk to your bot. Apologies for the inconvenience!!

**Comment by nsantini** [comment]
oh, I see, let me try that

**Comment by nsantini** [comment]
it worked!, thanks a lot @eanders-MS

**Comment by eanders-MS** [comment]
@nsantini The problem should be fixed now. It is safe to sync to the latest code. Incidentally, we also released a new emulator yesterday (version 3.5.31).
